AssemblerLib/MeshComponentMap.cpp

```diff
@@ -75,12 +100,23 @@ MeshComponentMap::MeshComponentMap(
                 // mesh items are ordered first by node, cell, ....
```

Comment:
Merged the common part with two if-conditions. Since this function is called once before the time stepping and the increasing of partitions decreases the size of the node loop, the two if-conditions inside the node loop should not slow down the computation distinctively.